control_panel_api/management/commands/load_kube_users.py

```diff
@@ -0,0 +1,42 @@
+import base64
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db.utils import IntegrityError
+from kubernetes import config, client
+
+from control_panel_api.models import User
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
So this would be available via `python manage.py load_kube_users`, nice :)!

I wonder if we should change `load_` with `import_` to be more explicit, maybe call it `import_k8s_users`? Just an idea.